### PR TITLE
Benchmark samples

### PR DESCRIPTION
**Pull request by dlam**

### CODE REVIEW

File: BasicRxJavaSample/versions.gradle [hunk 1]
```diff
@@ -15,19 +15,19 @@
...
-* Shared file between builds so that they can all use the same dependencies and
-* maven repositories.
-**/
+ * Shared file between builds so that they can all use the same dependencies and
+ * maven repositories.
+ **/
 ext.deps = [:]
 def versions = [:]
-versions.android_gradle_plugin = '3.3.1'
+versions.android_gradle_plugin = '3.4.0-rc02'
```
Comment (ChrisCraik) [inline BasicRxJavaSample/versions.gradle]: Use 3.4.0, now that 3.4 is stable

Comment (dlam) [reply BasicRxJavaSample/versions.gradle]: Done

**Comment by ChrisCraik** [comment]
LGTM, but I'll let Rahul review as well.

**Comment by dlam** [comment]
@tikurahul :D

**Pull request merged**